{
  try {
    throw new IdNotFoundException(objectName,pathToObject,objectType);
  }
 catch (  IdNotFoundException testedException) {
    assertEquals(expectedNullMessage,testedException.getMessage());
    assertEquals(null,testedException.getCause());
    assertEquals(objectName,testedException.getObjectName());
    assertEquals(pathToObject,testedException.getPathToObject());
    assertEquals(objectType,testedException.getObjectType());
  }
}
